test/integration/volume/attach_detach_test.go

```diff
@@ -73,11 +73,14 @@ func fakePodWithVol(namespace string) *v1.Pod {
 	return fakePod
 }
 
+type podCountFunc func(int) bool
+
 // Via integration test we can verify that if pod delete
 // event is somehow missed by AttachDetach controller - it still
 // gets cleaned up by Desired State of World populator.
 func TestPodDeletionWithDswp(t *testing.T) {
-	_, server, closeFn := framework.RunAMaster(nil)
+	masterConfig := framework.NewIntegrationTestMasterConfig()
+	_, server, closeFn := framework.RunAMaster(masterConfig)
```

Comment:
Yes please. Because it is an lvalue  right now I need to reason about it's usage later. Making it an rvalue improves readability.